theme_nano/views/new_page_template.xml

```diff
@@ -193,7 +184,13 @@
 
 <template id="new_page_template_about_personal_s_numbers" inherit_id="website.new_page_template_about_personal_s_numbers">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc2" remove="o_cc3" separator=" "/>
+        <attribute name="class" remove="o_cc3" separator=" "/>
```

Comment:
Doesn't doing this leave a remaining `o_cc` without `o_ccN` ? (several times in this commit - and, I never thought about checking that in the previous themes - maybe to be handled in a cleanup for already merged themes later)

Reply:
This one is a tricky one, I do agree 😅 

I do agree that with that `add=""` and `remove=""` it should be empty and thus `o_cc` but it seems to keep the value from the default template so `o_cc5`.

I made a previous try by adding `o_cc5` but runbot told me there were duplicated classes. By checking the new pages in the modal, it seems fine, but these templates should be refined later, indeed